src/utils/chain.ts

```
@@ -254,9 +254,20 @@ export async function createChatLunaChain(
             const chunkQueue = createAsyncChunkQueue<ChatLunaChainStreamChunk>()
+            const ctl = new AbortController()
             let buf = ''
             const toolCalls: ChatLunaChainStreamChunk['toolCalls'] = []
 
+            if (options?.signal?.aborted) {
+                ctl.abort()
+            } else {
+                options?.signal?.addEventListener(
+                    'abort',
+                    () => ctl.abort(),
+                    { once: true }
```

ESLint

[error] Replace `⏎····················'abort',⏎····················()·=>·ctl.abort(),⏎····················{·once:·true·}` with `'abort',·()·=>·ctl.abort(),·{⏎····················once:·true`

(prettier/prettier)